docs/user_utilities.rst

```diff
@@ -34,16 +34,18 @@ Based on spdlog, Taichi provides a series of APIs for logging. Note that these f
 .. code-block:: python
 
     import taichi as ti
-    ti.init(arch = ti.cpu)
+    ti.init(arch=ti.cpu)
 
     ti.set_logging_level(ti.INFO)
-    var = ti.var(ti.f32, shape = 1)
+    var = ti.var(ti.f32, shape=1)
+
 
     @ti.kernel
     def compute():
         var[0] = 1.0
         ti.info(f"set var[0] = 1.0")
```

Comment:
```suggestion
```
Let's be clear that `ti.info` cannot be used in Python-scope.
They should be moved to **developer utilities**, the logging system is mainly used by Taichi developers, not end-users.

Reply:
Nice, WIP

Reply:
> Let's be clear that `ti.info` cannot be used in Python-scope.

In this example, it seems `ti.info` is in Taichi-scope... :O?